simplq/public/index.html

```diff
@@ -26,6 +26,7 @@
       rel="stylesheet"
     />
     <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet" />
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" crossorigin="anonymous" />
```

Comment:
Why did we need this? We are pulling icons from Material UI, right?

Reply:
That's Right, But Material UI doesn't have an icon for MEDIUM. So for that, I have used font-awesome CDN

Reply:
Got it. Is it possible to download the vector image from font awesome or somewhere else and keep it in `public/images` and use it?

Rationale is that we don't want clients to download the entire awesome fonts assets, what do you think?

Reply:
Yes, this can be done too. Let me know if that's the way we are following. So I can make it done

Reply:
Yes, let do this. Download the asset and use it. We don't want to have two icon sets. Material UI icons also, I am planning to optimise later on.